templates/single-quiz.php

```diff
@@ -86,11 +92,13 @@ class="<?php sensei_the_question_class(); ?>"
 						<?php
 
 						/**
-						 * Action inside before the question content on single-quiz page
+						 * Action inside before the question content on single-quiz page.
 						 *
-						 * @hooked Sensei_Question::the_answer_feedback
+						 * @hook sensei_quiz_question_inside_after
+						 *
+						 * @param {int} $the_question_id The question ID.
```

Comment:
```suggestion
						 * @param {int} $question_id The question ID.
```

Same as the above comment.